### ISSUE TITLE
Add from to sms.ru

### ISSUE THREAD

**Issue by bigperson** [nutnetru/laravel-sms#7]
How to add parameter from?

**Comment by maximkou** [comment]
Hi!

At this moment, package supports only passing this parameter at send time, to example:

```php
$sender->send('<phone_number>', '<your_message>', [
    'from' => '<sender name here>'
]);
```

All options from this [list](http://sms.ru/api/send) is supported when using this way.

**Comment by maximkou** [comment]
@bigperson Hi!

Just now i'm released version 0.8.0, which added ability to defining provider-wide parameters.
Now you can define default message parameters in config, for example:

```php
// config/nutnet-laravel-sms.php
'providers' => [
    'smsru' => [
        // your other settings
        'message_defaults' => [
            'from' => '<sender name>'
        ]
    ]
],
```

and you can send messages simply:

```php
// "from" parameter will be added automatically, if defined in "message_defaults" option in config
$sender->send('<phone_number>', '<your_message>');
```

**Attention:** configuration file format is changed, if you update to 0.8.0, dont forget to refresh it.
